src/Symfony/Bridge/Twig/Mime/WrappedTemplatedEmail.php

```diff
@@ -62,10 +62,7 @@ public function attach(string $file, string $name = null, string $contentType =
         }
     }
 
-    /**
-     * @return $this
```

Comment:
Acutally, this is not the same.
* $this means it's exactly the same object
* self means it's may be another instance of this class

Reply:
yes please add the return type but keep the phpdoc

Reply:
We need to settle on a policy here.
Personally, I feel link using self is more wrong than correct, so using `object` + the annotation might more accurate.
But that's just my opinion :)

Reply:
`: self` is as close as we can get to `@return $this` at the moment. However, as long as php does not support covariant return types, using `: self` here might create awkward situations when overriding such a method or when moving it to a superclass.

Reply:
I think typhinting `object` for `$this` is bad. For IDEs that don't support $this, having an object return type means nothing. It does not provide typesafety as would `self` do in most cases. So for users, self is much better.
The argument was it can provide awkward situation when overriding. But this PR is about final classes which should not be overridden. So again would only affect us maintainers and PHP would complain about it directly in PHP <7.4. So not a problem.
So I'm in favor of using `self` which is the closest to `$this`.